### PR TITLE
chore(iOS): update for Titanium 10.0.1

### PR DESCRIPTION
**Pull request by jquick-axway**
**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28458

**Summary:**
- Replaced usage of getter/setter property methods.
- Replaced usage of deprecated `2DMatrix` with `Matrix2D`.
- Removed usage of deprecated `UILocalNotification`.
- Removed unused imports.
- Fixed JS code indentation.

**Note:**
This app will crash on startup on iOS due to a Titanium 10.0.0 regression with the `Hyperloop.defineClass()` method. Will be fixed by the following PR to be fixed in Titanium 10.0.1.
https://github.com/appcelerator/titanium_mobile/pull/12829

### CODE REVIEW

**Review by ewanharris — CHANGES_REQUESTED** [review]
Tested across iOS (using appcelerator/titanium_mobile/pull/12829) and Android, looks good!

Just one comment on the `ios/gravity` example where it looks like the import has changed

File: app/controllers/ios/gravity.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
-import { UIDynamicAnimator, UIDynamicItemBehavior, UIGravityBehavior, UICollisionBehavior, UIColor } from 'UIKit';
+import { UIDynamicAnimator, UIDynamicItemBehavior, UIGravityBehavior, UICollisionBehavior } from 'UIKit';
 import { NSBundle, NSURL } from 'Foundation';
 import { AVAudioPlayer } from 'AVFoundation';
```
[the remark this answers is not in the document]
Comment (ewanharris) [reply app/controllers/ios/gravity.js]: Figured it just made more sense to push this to your branch seeing as I had it checked out and edited

**Review by ewanharris — APPROVED** [review]
Changing to approval

File: app/controllers/ios/gravity.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jquick-axway) [reply app/controllers/ios/gravity.js]: Per what we discussed, this looks like a breaking-change Apple made in the 14.5 iOS SDK. Changing the import from `AVFoundation` to `AVFAudio` will break for anyone using 14.4 (like me). But we need to move forward and make this change. Thanks Ewan.

**Comment by lokeshchdhry** [comment]
FR Passed.
Checked with 10.0.1 SDK

**Pull request merged**